### PR TITLE
Add support for CSSModules and SASS

### PR DESCRIPTION
**Pull request by stereobooster**
Tests failing, but not because of this PR

```
 FAIL  src/App.test.js
  ● Test suite failed to run

    /Users/slavik/my/create-react-app/packages/react-scripts/template/src/App.less:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){.App {
                                                                                             ^
    SyntaxError: Unexpected token .

      at transformAndBuildScript (../node_modules/jest-runtime/build/transform.js:284:10)
      at Object.<anonymous> (src/App.js:5:27)
      at Object.<anonymous> (src/App.test.js:3:38)

Test Summary
 › Ran all tests.
 › 1 test suite failed, 0 tests passed (0 total in 1 test suite, run time 1.609s)
```

Still looking for solution

### CODE REVIEW

**Comment by stereobooster** [comment]
I converted to ES6, because recently used it, but on the second thought maybe it wasn't that bright idea. Can revert to ES5.

@kitze how do you run tests?

**Comment by A-gambit** [comment]
@kitze Why you ignore this pull request? Is there any problem?

**Comment by kitze** [comment]
@stereobooster can you please pull latest changes and resolve the conflicts? Then I can merge this.

**Comment by stereobooster** [comment]
@kitze rebased

File: packages/react-scripts/config/get-custom-config.js [hunk 1]
```diff
@@ -1,21 +1,26 @@
...
-  var prod = prod || false;
-  var env = env || {};
-  var result = Object
-    .keys(customizers)
-    .reduce(function (finalConfig, customizerKey) {
-      var customizer = customizers[customizerKey];
+  prod = prod || false;
+  const env = {};
+
+  const customizerKeys = customizers.keys();
```
Comment (karlhorky) [inline packages/react-scripts/config/get-custom-config.js]: Doesn't this need to be the following?

```
const customizerKeys = Object.keys(customizers);
```

I tried out your fork here, and it failed on that line. I fixed it in karlhorky/custom-react-scripts#3ad34e8930dc4d72c68622714c910c16a6f4b27d

**Comment by karlhorky** [comment]
Looks like there's conflicts again too.

**Comment by A-gambit** [comment]
@stereobooster pull master and fix bug, plz

**Comment by sampi** [comment]
@stereobooster I think this PR should fix the tests https://github.com/kitze/create-react-app/pull/52

**Comment by karlhorky** [comment]
As an alternative, you can also try out my fork: https://github.com/karlhorky/create-react-app#readme

* CSS Modules
* PostCSS (with imports, variables, and nesting)
* Style linting with `stylelint`
* Airbnb JavaScript linting

**Comment by kitze** [comment]
This will be done in another way.

**Pull request closed without merging**